AppBuilder/platform/views/viewComponent/ABViewPivotComponent.js

```diff
@@ -71,6 +71,18 @@ module.exports = class ABViewPivotComponent extends ABViewComponent {
                   }
                },
             ],
+            [
+               pivot.views.table,
+               class CustomTable extends pivot.views.table {
+                  CellFormat(value) {
+                     const decimalPlaces = settings.decimalPlaces ?? 2;
+                     if (!value) value === 0 ? "0" : "";
```

Comment:
Do we not need this line ?

Reply:
I just copied from webix `Table.CellFormat()`. But yes, it's not needed.